### PR TITLE
MAUI-3667-CustomizationUGContent_MAUI_SfCalendar

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
## **Description**

- Added the UG content for Customization of Year view, Month view

- Included the Selection background and Text style for Month view and Year view

### CODE REVIEW

File: MAUI/Calendar/customizations.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+documentation: ug
+---
+
+# Customizations in MAUI Calendar (SfCalendar)
+Each cell of the Month, Year, Decade and Century can be customized in MAUI Calendar (SfCalendar).
+
+## Month cell customization
+You can customize the calendar month view cell by using the `MonthView` property of `SfCalendar`.
+
+*    **Month dates** – You can customize the month date's text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `MonthView`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:16]: You can customize the month dates' text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `MonthView.`

File: MAUI/Calendar/customizations.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+*    **Today date** – You can customize the today date text style and background of the `Calendar` by using the `TodayTextStyle` and `TodayBackground` properties of `MonthView`.
+
+*    **Trailing and Leading dates** – You can hide the trailing and leading date by using the `ShowTrailingAndLeadingDates` property in the `CalendarMonthView` class. You can also customize the Trailing and Leading dates text style and background of the `Calendar` by using the `TrailingLeadingDatesTextStyle` and `TrailingLeadingDatesBackground` properties of `MonthView`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:20]: You can hide the trailing and leading dates by using the `ShowTrailingAndLeadingDates` property in the `CalendarMonthView` class. You can also customize the trailing and leading dates text style and background of the `Calendar` by using the `TrailingLeadingDatesTextStyle` and `TrailingLeadingDatesBackground` properties of `MonthView.`

File: MAUI/Calendar/customizations.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+*    **Disabled dates** – You can disable the date by using the MinimumDate, MaximumDate, EnablePastDates and SelectableDayPredicate callback. The date before `MinimumDate` is said to disabled date, the date after the `MaximumDate` is said to disabled date, if you want disable the date before today date by using the `EnablePastDates` and if you want to disable any particular date by using `SelectableDayPredicate` property. You can also customize the disabled dates text style and background of the `Calendar` by using the `DisabledDatesTextStyle` and `DisabledDatesBackground` properties of `MonthView`.
+
+*    **Special Dates** – You can add special dates to the `DateRangePicker` by using `SpecialDates` property and you can customize the special dates textstyle and background of the `Calendar` by using the `SpecialDatesTextStyle` and `SpecialDatesBackground` properties of `MonthView`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:24]: You can add special dates to the `DateRangePicker` by using the `SpecialDates` property and you can also customize the special dates text style and background of the `Calendar` by using the `SpecialDatesTextStyle` and `SpecialDatesBackground` properties of `MonthView.`

File: MAUI/Calendar/customizations.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+*    **Weekend Dates** – You can customize the weekend dates textstyle and background of the `Calendar` by using the `WeekendDatesTextStyle` and `WeekendDatesBackground` properties of `MonthView`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:26]: You can customize the weekend dates text style and background of the `Calendar` by using the `WeekendDatesTextStyle` and `WeekendDatesBackground` properties of `MonthView.`

File: MAUI/Calendar/customizations.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+        this.Calendar.MonthView.WeekendDatesBackground = Colors.Red;
+        this.Calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-customization-in-maui-calendar](images/customization/monthview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading Dates and weekend dates.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:93]: The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading dates, and weekend dates.

File: MAUI/Calendar/customizations.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+    TextFormat = "ddd",
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![view-header-text-format-in-maui-calendar](images/customization/view-header-text-format-in-maui-calendar.png)
+
+## Year cell customization
+You can customize the calendar `year`, `decade` and `century` view by using the `YearView` property of `SfCalendar`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:126]: You can customize the calendar `year`, `decade,` and `century` view by using the `YearView` property of `SfCalendar.`

File: MAUI/Calendar/customizations.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+![decadeview-customization-in-maui-calendar](images/customization/decadeview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, disable dates, today date and leading Dates.
+
+## Year View Text format
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:187]: Year view text format

File: MAUI/Calendar/customizations.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+
+*   **Year cell dates** – You can customize the year view date's text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `YearView`.
+
+*    **Disabled dates** – You can disable the date by using the MinimumDate, MaximumDate, EnablePastDates and SelectableDayPredicate callback. The date before `MinimumDate` is said to disabled date, the date after the `MaximumDate` is said to disabled date, if you want disable the date before today date by using the `EnablePastDates` and if you want to disable any particular date by using `SelectableDayPredicate` property. You can also customize the disabled dates text style and background of the `Calendar` by using the `DisabledDatesTextStyle` and `DisabledDatesBackground` properties of `YearView`.
+
+*    **Leading dates** – You can hide the Leading date by using the `ShowTrailingAndLeadingDates` property in the `SfCalendar` class. You can also customize the Leading dates text style and background of the `Calendar` by using the `LeadingDatesTextStyle` and `LeadingDatesBackground` properties of `YearView`. It is not applicable for Year view.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:132]: You can hide the leading dates by using the `ShowTrailingAndLeadingDates` property in the `SfCalendar` class. You can also customize the leading dates text style and background of the `Calendar` by using the `LeadingDatesTextStyle` and `LeadingDatesBackground` properties of `YearView.`

File: MAUI/Calendar/customizations.md [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![yearview-text-format-in-maui-calendar](images/customization/yearview-text-format-in-maui-calendar.png)
+
+## Selection cell customization
+
+You can customize the Selection cell background and textStyle in Month, Year, Decade and Century view by using the `SfCalendar`.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/customizations.md:215]: You can customize the selection cell background and text style in Month, Year, Decade, and Century view by using the `SfCalendar.`

File: MAUI/Calendar/customizations.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+        this.calendar.MonthView.WeekendDatesBackground = Color.FromArgb("#E2F9F3");
+        this.calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view Customization in .NET MAUI Calendar.](images/customization/maui-month-view-customization.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading dates, and weekend dates.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/customizations.md:94]: Check the priority order on local

File: MAUI/Calendar/customizations.md [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+
+## Month View Header format
```
Comment (Sasikumar3595) [inline MAUI/Calendar/customizations.md:96]: Specify this view header format, and header format under header topic

File: MAUI/Calendar/customizations.md (hunk 10 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Checked and changed the order

File: MAUI/Calendar/customizations.md (hunk 11 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content modified

File: MAUI/Calendar/customizations.md [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+layout: post
+title: Customizations in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about Customizations features of Syncfusion .NET MAUI Calendar (SfCalendar) control and more. 
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Customizations in .NET MAUI Calendar (SfCalendar)
+Each cell of the Month, Year, Decade and Century can be customized in .NET MAUI [Calendar](https://www.syncfusion.com/maui-controls/maui-calendar) (SfCalendar).
```
Comment (Sasikumar3595) [inline MAUI/Calendar/customizations.md:11]: .NET MAUI [Calendar] change into [.NET MAUI Calendar]

File: MAUI/Calendar/customizations.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![Month view Customization in .NET MAUI Calendar.](images/customization/maui-month-view-customization.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, weekend dates, trailingLeading dates, and normal dates.
+
+## Year cell customization
+You can customize the calendar `year`, `decade`, and `century` view by using the `YearView` property of `SfCalendar`.
```
Comment (Sasikumar3595) [inline MAUI/Calendar/customizations.md:97]: You can customize the calendar `year`, `decade`, and `century` view**s**

File: MAUI/Calendar/customizations.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 3 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 5 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 6 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 7 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 8 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 9 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content changed

File: MAUI/Calendar/customizations.md (hunk 13 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Content modified

File: MAUI/Calendar/customizations.md [hunk 14]
```diff
@@ -0,0 +1,219 @@
...
+            FontSize = 12,
+        };
+
+        this.Calendar.View = CalendarView.Decade;
+        this.Calendar.MinimumDate = DateTime.Now.AddYears(-1);
+        this.Calendar.MaximumDate = DateTime.Now.AddYears(8);
+        this.Calendar.EnablePastDates = false;
+        this.calendar.Background = Colors.PaleGreen.WithAlpha(0.3f);
+        this.calendar.YearView.TextStyle = textStyle;
+        this.calendar.YearView.TodayBackground = Colors.Pink;
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/customizations.md]: Modified the Code

**Review by Nijamudeen-Mohamed — APPROVED** [review]

**Pull request merged**